### PR TITLE
[AutorunMediaJob] make hardcoded strings translatable

### PR DESCRIPTION
**Pull request by un1versal**
Like title says.

@Paxxi knows. :)

### CODE REVIEW

**Comment by ksooo** [comment]
@uNiversaI Thx, this one was bugging me for a long time.

**Comment by MartijnKaijser** [comment]
should be backported as well

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -11432,7 +11432,30 @@ msgctxt "#21330"
 msgid "Show hidden files and directories"
 msgstr ""
 
-#empty strings from id 21331 to 21335
+#: xbmc/storage/AutorunMediaJob.cpp
```
Comment (da-anda) [inline addons/resource.language.en_gb/resources/strings.po]: mind adding contextual help strings?

**Comment by da-anda** [comment]
I've never seen this dialog - how to trigger? It didn't show up when I plugged a USB-Stick or HDD. I'm asking because I think the dialog might look a bit cryptic and could probably need a little more text, like "what would you like to do?"

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (uNiversaI) [inline addons/resource.language.en_gb/resources/strings.po]: I do, they dont belong here. Its not that type of dialog.

**Comment by uNiversaI** [comment]
Its shows up when you insert optical media here for instance. HDD/USB no for some reason but its should. I cant fix or extend that anyway.

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (da-anda) [inline addons/resource.language.en_gb/resources/strings.po]: still strings could be reused by devs in future and thus a contextual comment would help. It also helps translators to know if a string is f.e. a dialog headline, message text or a settings value (f.e. to adjust length of translation)

**Comment by da-anda** [comment]
it's not triggered on Windows when I insert a CD with pictures. All I get is "Play disc" showing up in the main menu, which is totally wrong for this particular disc. I don't even get the kaitoast when un-/plugging USB media like I do on OpenElec. Can somebody on Windows confirm that nothing at all is happening, or is it just my test setup?

**Comment by uNiversaI** [comment]
Yea even the kaytoasts is all messed up here in Linux, on CD/insert it says I "inserted removable device" not a CD (internal CD/DVD not usb). No surprise there for me all external media handling most notably optical media has barely worked so far.

**Comment by MartijnKaijser** [comment]
jenkins build and merge

**Pull request merged**